<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SyntroCVFilterClass</class>
 <widget class="QMainWindow" name="SyntroCVFilterClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SyntroCVFilter</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="actionBasicSetup"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionVideoStreams"/>
    <addaction name="actionShowName"/>
    <addaction name="actionShowDate"/>
    <addaction name="actionShowTime"/>
    <addaction name="actionTextColor"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuActions"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionExit"/>
   <addaction name="separator"/>
   <addaction name="actionVideoStreams"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionBasicSetup">
   <property name="text">
    <string>Basic Setup</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionVideoStreams">
   <property name="text">
    <string>Video streams ...</string>
   </property>
   <property name="iconText">
    <string>Video Streams</string>
   </property>
  </action>
  <action name="actionShowName">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show name</string>
   </property>
  </action>
  <action name="actionShowDate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show date</string>
   </property>
  </action>
  <action name="actionShowTime">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show time</string>
   </property>
  </action>
  <action name="actionTextColor">
   <property name="text">
    <string>Text color ...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="syntrocvfilter.qrc"/>
 </resources>
 <connections/>
</ui>
